drag-and-drop/library/src/utils/runtime-flow-state.ts

```diff
@@ -188,6 +191,105 @@ export class RuntimeFlowState {
     asset.value = undefined;
   }
 
+  createDropTarget(
+    pluginName: string,
+    targetAssetType: ObjectType,
+    targetAsset: Asset,
+    propName = '',
+    parentAssetType = ''
+  ): DropTargetAssetType {
+    const dropTarget: DropTargetAssetType = {
+      id: uuidv4(),
```

Comment:
Since we're loading existing content you can probably derive a drop target name from the asset that is being wrapped to avoid adding a new dependency.